packages/website/data/sponsors.json

```diff
@@ -244,19 +252,27 @@
     "website": "https://backscattering.de"
   },
   {
-    "id": "Corellium",
-    "image": "https://images.opencollective.com/corellium/aa8c228/logo.png",
-    "name": "Corellium",
+    "id": "Christian Healthcare Ministries",
+    "image": "https://images.opencollective.com/christian-healthcare-ministries/5f05f8c/logo.png",
+    "name": "Christian Healthcare Ministries",
     "tier": "contributor",
-    "totalDonations": 4800,
-    "website": "https://www.corellium.com"
+    "totalDonations": 2100,
+    "website": "https://sharehealthcare.com"
```

Comment:
I am personally tentatively opposed to including a link to Christian Healthcare Ministries because of their stance on abortions. CHS and other Christian "healthcare cost solution"s explicitly don't cover abortion. They instead encourage pregnant members go to a "Christian pregnancy center" [[general guidelines](https://www.chministries.org/media/5189/chmguidelines.pdf) / [more specific guidelines](https://f.hubspotusercontent20.net/hubfs/6634526/chmguidelines-1.pdf)].

But I can understand if this is too overreaching as an opinion (e.g. plenty of Christian people do support the right to choose).

I've started a thread in a private maintainers channel to discuss. If you have strong opinions here, please let me know privately via the email on my profile.